x-pack/plugins/ml/public/settings/_index.scss

```diff
@@ -1,3 +1,3 @@
 @import 'settings';
 @import 'filter_lists/index';
-@import 'scheduled_events/index';
\ No newline at end of file
+@import 'calendar/index';
```

Comment:
This needs to be `calendars/index` to pick up the renamed `calendars` directory.